src/Domain/DomainHelpers.cpp

```diff
@@ -401,6 +461,39 @@ void set_internal_boundaries(
   }
 }
 
+template <size_t VolumeDim>
+void set_internal_boundaries(
+    const std::vector<std::unique_ptr<domain::CoordinateMapBase<
+        Frame::Logical, Frame::Inertial, VolumeDim>>>& maps,
+    gsl::not_null<
```

Comment:
`const` and we prefer `notnull, std::vector` order, though I think that's inconsistent in the existing function of this name so I'm fine filling an issue and addressing it in both functions later